orchestra/templates/orchestra/base.html

```diff
@@ -75,6 +75,16 @@
     {% block main %}
     {% endblock %}
 
+    {% block static_url %}
+      <script type="text/javascript">
+        // NOTE(joshblum): This is a noop function that is replaced by django static file
+        // processing, but allows us to work in dev.
+        function $static(url) {
```

Comment:
my javascript-foo isn't strong enough to know, but should this be:

`window.$static = function(url) {...`